src/main/java/com/prgrms/commandLineApplication/voucher/FixedAmountVoucher.java

```diff
@@ -0,0 +1,20 @@
+package com.prgrms.commandLineApplication.voucher;
+
+public class FixedAmountVoucher extends Voucher {
+
+  public FixedAmountVoucher(String voucherType, double discountAmount) {
+    super(voucherType, discountAmount);
+  }
+
+  public double discount(double price) {
+    validateDiscountAmount(getDiscountAmount());
+    return price - getDiscountAmount();
+  }
```

Comment:
```suggestion
  public double discount(double price) {
    validateDiscountAmount(price);
    return price - getDiscountAmount();
  }
```
로직이 제대로 작성되지 않은 것 같아요. 그리고 멤버 변수에 대한 접근을 getter로 할 필요가 있을까요?

Reply:
이 부분은 discountAmount가 `Voucher`의 인스턴스 변수이고, private으로 설정되어 있어 그런 것 같습니다. 
`price`가 아닌 `discountAmount`에 대한 유효성 검증 로직인 것 같아요.  `discount()`에 있을 것이 아니라 `FixedAmountVoucher`에 있어야 할 로직인 것 같네요

Reply:
2caab08459e818932edaca263413daf7a4a43df3

Reply:
커밋 후 추가되지 않은 부분이 있어서 'commit —amend -m' 으로 추가했더니 아직 구현하지 않은 다른 클래스들도 같이 올라가버렸습니다🥺 이전 피드백 반영할 때도 이런 상황이 생겼었는데 이유를 이제야 알게되었습니다 주의하겠습니다🥲🥲☀️